lib/elixir/lib/access.ex

```diff
@@ -422,4 +422,58 @@ defmodule Access do
   defp all([], _next, gets, updates) do
     {:lists.reverse(gets), :lists.reverse(updates)}
   end
+
+  @doc ~S"""
+  Accesses the _nth_ element (zero based) of a list.
```

Comment:
It doesn't make sense to say `nth` and zero based. You probably want to say: "Accesses the element at `index` (zero based) of a list".